sopel/plugins/handlers.py

```diff
@@ -435,7 +531,13 @@ class EntryPointPlugin(PyModulePlugin):
         .. __: https://setuptools.readthedocs.io/en/stable/setuptools.html#dynamic-discovery-of-services-and-plugins
 
     """
+
     PLUGIN_TYPE = 'setup-entrypoint'
+    """Plugin's type.
+
+    Metadata for the plugin, this should be considered to be a constant and
+    should not be modified at runtime.
+    """
```

Comment:
```suggestion
    """The plugin's type.

    Metadata for the plugin; this should be considered to be a constant and
    should not be modified at runtime.
    """
```

Ditto note for `PLUGIN_TYPE` constant in `PyModulePlugin`.